keyboards/anavi/knob1/info.json

```
@@ -0,0 +1,51 @@
+{
+    "keyboard_name": "Knob 1",
+    "manufacturer": "ANAVI",
+    "url": "https://github.com/AnaviTechnology/anavi-knob-1",
+    "maintainer": "leon-anavi",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "debounce": 5,
+    "features": {
+        "bootmagic": false,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": true,
+	"rgblight": true
+    },
+    "rgblight": {
+        "pin": "GP12",
+        "led_count": 1,
+	"hue_steps": 10,
+	"saturation_steps": 17,
+        "brightness_steps": 17,
+	"max_brightness": 255,
+	"animations": {
+            "all": true
```

Could you explicitly define which animations are enabled? 
```suggestion
            "alternating": true,
            "breathing": true,
            "christmas": true,
            "knight": true,
            "rainbow_mood": true,
            "rainbow_swirl": true,
            "rgb_test": true,
            "snake": true,
            "static_gradient": true,
            "twinkle": true
```

---

Thank you for the code review. Done. I've forced pushed the modified commit. Thanks, Leon